docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_title: Troubleshooting
+---
+### Summary
+
+The following page provides suggestions for common errors that may occur during firmware compilation. If the information provided is insufficient to resolve the issue, feel free to seek out help from the [ZMK Discord](https://zmkfirmware.dev/community/discord/invite).
+
+### Environment Variables not Defined
+
+An error along the lines of `CMake Error at (zmk directory)/zephyr/cmake/generic_toolchain.cmake:64 (include): include could not find load file:` during firmware compilation indicates that the Zephyr Environment Variables are not properly defined.
+For more information, click [here](/docs/dev-setup#environment-variables).
+
+### Improperly defined keymap
```

Comment:
Same suggestion, perhaps `### DTS (devicetree) Errors` here to start w/ the problem, not the root cause/solution?